docs/setup-and-installing.md

```diff
@@ -4,11 +4,12 @@
 
 # Set up and get started with the SpatialOS Unity GDK
 
-**Note:** The Unity GDK currently runs on Windows only but a MacOS version is to come.
-
 ## Short version (for SpatialOS local deployment only)
 
-1. Set up your machine by installing: [Unity 2018.2.0b10](https://unity3d.com/unity/beta-download), [Visual Studio 2017](https://www.visualstudio.com/downloads/), [SpatialOS](https://console.improbable.io/installer/download/stable/latest/win), [ReSharper](https://www.jetbrains.com/resharper/) (optional), and [ReSharper Command Line Tools](https://www.jetbrains.com/resharper/download/index.html#section=resharper-clt) (optional).
+1. Set up your machine by installing: 
+* (For Windows) [Unity 2018.2.0b10](https://unity3d.com/unity/beta-download), [Visual Studio 2017](https://www.visualstudio.com/downloads/), [.NET Core 2.1](https://www.microsoft.com/net/download/), [SpatialOS](https://console.improbable.io/installer/download/stable/latest/win), [ReSharper](https://www.jetbrains.com/resharper/) (optional), and [ReSharper Command Line Tools](https://www.jetbrains.com/resharper/download/index.html#section=resharper-clt) (optional)
+* (For Mac) [Unity 2018.2.0b10](https://unity3d.com/unity/beta-download), [Rider](https://www.jetbrains.com/rider/), [.NET Core 2.1](https://www.microsoft.com/net/download/), [SpatialOS](https://console.improbable.io/installer/download/stable/latest/win)
```

Comment:
Rider should be optional as it is not a free software

Reply:
There's also Visual Studio for Mac, which Unity supports and is free.

Reply:
Is it similar to what we've said about ReSharper for Windows? I.e. is Rider optional unless you want to contribute to the repo, in which case you need it because of our linting rules?

Reply:
kind of, yes